.eslintignore

```diff
@@ -0,0 +1,5 @@
+tests
+package.json
+node_modules
+*.css
+*.md
```

Comment:
please add empty line in the end of file

Reply:
Yes, thanks. I forgot that prettier don't fix this type of file.